services/core-web/src/components/dashboard/customHomePage/SubscriptionTable.js

```diff
@@ -71,10 +70,7 @@ export class SubscriptionTable extends Component {
         render: (text, record) => (
           <div title="Permit Number">
             <ul className="mine-list__permits">
-              {text &&
-                uniqBy(text, "permit_no").map(({ permit_no, permit_guid }) => (
-                  <li key={permit_guid}>{permit_no}</li>
-                ))}
+              <p>{text.map((permit_no) => [<span>{permit_no}</span>, <br />])}</p>
               {!text && <li>{record.emptyField}</li>}
```

Comment:
this should have a 'key' attribute, it's not breaking code but will add errors in the console.